caiman/base/movies.py

```diff
@@ -226,6 +306,26 @@ def bin_median(self, window=10):
         num_frames = num_windows * window
         return np.nanmedian(np.nanmean(np.reshape(self[:num_frames], (window, num_windows, d1, d2)), axis=0), axis=0)
 
+    def bin_median_3d(self, window=10):
+        """ compute median of 4D array in along axis o by binning values
+
+        Args:
+            mat: ndarray
+                input 4D matrix, time along second dimension
+
+            window: int
+                number of frames in a bin
+
+        Returns:
+            img:
+                median image
+
+        """
+        d1, T, d2, d3 = np.shape(self)
```

Comment:
We typically have the movie ordered as T, X, Y (,Z) with time in the first dimension. This is also what the CNMF expects. Is there any reason for this choice?

Reply:
NWB also expects T, X, Y (,Z)